pkg/apis/metal3/v1alpha1/zz_generated.openapi.go

```diff
@@ -20,7 +20,6 @@ func schema_pkg_apis_metal3_v1alpha1_BareMetalHost(ref common.ReferenceCallback)
 		Schema: spec.Schema{
 			SchemaProps: spec.SchemaProps{
 				Description: "BareMetalHost is the Schema for the baremetalhosts API",
-				Type:        []string{"object"},
```

Comment:
Why did this get removed? If the generated code was out of date, could we handle fixing that separately?

Reply:
Everybody has a different version of the code generation tools installed and it bounces around on each PR :(

Reply:
It sounds like we need a CI job that installs the version we support and verifies that no changes are produced by regenerating.

Reply:
Or vendor all the things <shudder>